common/metadata-helper.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <string>
+
+namespace rs2
+{
+    // Helper class that is responsible for enabling per-frame metadata on different platforms
+    // Currently implemented for Windows
+    class metadata_helper
+    {
+    public:
+        static metadata_helper& instance();
+
+        // Check if metadata is enabled using Physical Port ID
+        virtual bool is_enabled(std::string id) const { return true; }
+
+        // Enable metadata for all connected devices
+        virtual void enable_metadata() { }
```

Comment:
Technically, it could fail, so I'd suggest to return a bool, even if the implementation always "succeeds"